main.js

```diff
@@ -55,4 +55,4 @@ function updateTwitterIntent(quote) {
   var intentAddress = 'https://twitter.com/intent/tweet?text=';
```

Comment:
Hmm, I'm a little unsure how to interpret the support scores on caniuse.com. Maybe "89% supported" isn't great for production code, but for what I'm doing it's probably okay. The only people I expect to look at this are most likely going to be using modern browsers. If I were trying to make this available to a wider audience I would probably avoid using it. 

BUT, I personally see a lot of value in "pretending" to work under these types of constraints, so I think I would prefer to explore better-supported options.

I suppose the other thing to consider is whether this aspect of the app is essential, which it isn't. But if `.animate()` isn't supported on the user's browser, I think the quotes won't render correctly, and the app will actually look kind of "broken", so that's not great.

I think I'm going to try to implement the animation in a better way.

Reply:
❤️